pom.xml

```diff
@@ -48,8 +48,7 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<source>${java.required}</source>
-					<target>${java.required}</target>
+					<release>${java.required}</release>
```

Comment:
@awildturtok bevor ich E2E bei mir lokal wieder zum Laufen bringen konnte hatte ich Probleme das Backend zu kompilieren.

In den Maven-Logs mit Java 23 gabs eine Warning à la: Lieber `release` nutzen als `source`+`target` da sonst nicht sicher gestellt wird, dass die richtigen System-Libs genutzt werden.

Durch das ändern war mein eigentlicher Fehler zwar nicht weggegangen, aber eine Warning weniger :)